packages/text-input-react/src/index.ts

```diff
@@ -1,5 +1,2 @@
-export { TextField } from "./TextField";
 export { TextArea } from "./TextArea";
-export { ActionTextField } from "./ActionTextField";
-export { InlineTextField } from "./InlineTextField";
-export { BaseInputField } from "./BaseInputField";
+export { TextInput } from "./TextInput";
```

Comment:
Intoduserer vi ikke problemer for AHV ved å fjerne BaseInput. De var avhengige å splitte Input og Label, er det addresert i TextInput nå?

Reply:
Godt poeng. Jeg tenkte at inline-versjonen ville dekke behovet, men innser at det kanskje ikke blir tilfelle likevel, nei.